src/style.css

```diff
@@ -0,0 +1,84 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir.ttc");
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.search-big-one {
+  margin: 20px 0;
+}
+
+.big-search-input {
+  font-family: Avenir, sans-serif;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 70px;
+  padding: 0 62px;
+  outline: none;
+  font-weight: 300;
+  font-size: 16px;
+}
+
+.small-search-input {
+  font-family: Avenir, Helvetica, sans-serif;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  width: 100%;
+  height: 42px;
+  padding: 0 33px;
+  outline: none;
+  font-weight: 300;
+  font-size: 14px;
+}
```

Comment:
These classes share a lot of same styles, It'd be better to combine these styles in one class search-input, and move different styles to modifications search-input--big and search-input--small